docs/content/any/project/changelogs/2021-05-17.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Release 2021-05-17
+menuTitle: 2021-05-17
+any: true
+description: >-
+  Changelog for Release 2021-05-17 (sqlalchemy-oso 0.6.2, sqlalchemy-oso-preview
+  0.0.3) containing new features, bug fixes, and more.
+---
+
+## `sqlalchemy-oso` 0.6.2
+
+### Other bugs & improvements
+
+- Authorized sessions now disable [baked queries][] by default because the
+  caching mechanism can bypass authorization by using queries from the cache
+  that were previously baked without authorization applied. If you understand
+  the risks and still want to opt-in to the previous behavior of using baked
+  queries, you can pass the `enable_baked_queries=True` keyword argument to
+  `sqlalchemy_oso.authorized_sessionmaker()` and friends.
+
+[baked queries]: https://docs.sqlalchemy.org/en/13/orm/extensions/baked.html
+
+## `sqlalchemy-oso-preview` 0.0.3
+
+### Breaking changes
+
+{{% callout "Warning" "orange" %}}
+  This release contains breaking changes. Be sure to follow migration steps
+  before upgrading.
+{{% /callout %}}
+
+#### Replaced `action` parameter with `checked_permissions`
```

Comment:
Have the docs been updated?

Reply:
Which docs? I updated doc strings so the sqlalchemy-oso API docs will reflect this change.

Reply:
This is gonna require new-roles docs updates, but I was waiting to do that until I've updated GitClub so that the code snippets are up to date